snap/snapcraft.yaml

```diff
@@ -204,12 +206,11 @@ plugs:
     interface: content
     content: aziot-identity-service
     target: $SNAP_DATA
+    allow-auto-connection: true
 
   account-control:
     interface: account-control
-
-  hardware-observe:
```

Comment:
After removing this plug, please ensure that all DeviceInfo_* functions are still functioning properly.